src/components/Breadcrumbs.js

```diff
@@ -22,10 +22,17 @@ export default class Breadcrumbs extends Component {
       const paths = splat.split('/');
       links = _.map(paths, (path, i) => {
         const before = (i == 0) ? '' : paths.slice(0, i).join('/') + '/';
-        return {
-          href: `${base}/${before}${path}`,
-          label: path
-        };
+        if (type == "data files") {
```

Comment:
But we defined our routes so that individuals are called with their extensions. 

```ruby
get "/*?/?:path.?:ext?"
```

`/_api/data/movies` returning for `movies.yml` file doesn't make sense for me. IMO we need to be as specific as possible while calling the endpoints.